tests/test_hubspot_interrupted_sync.py

```diff
@@ -38,6 +38,12 @@ def simulated_interruption(self, reference_state):
         new_state['bookmarks']['engagements']['lastUpdated'] = None
         new_state['bookmarks']['engagements']['current_sync_start'] = engagements_bookmark
 
+        tickets_bookmark = self.timedelta_formatted(
+            reference_state['bookmarks']['engagements']['lastUpdated'],
+            days=-1, str_format=self.BASIC_DATE_FORMAT
+        )
```

Comment:
```suggestion
        tickets_bookmark = self.timedelta_formatted(
            reference_state['bookmarks']['engagements']['lastUpdated'],
            days=-1, str_format=self.BASIC_DATE_FORMAT)
```

Reply:
Also why are we handlling `tickets` stream separately?

Reply:
we setting the bookmark value for tickets stream in this way. Followed the existing pattern.